docs/en/05_Contributing/02_Request_for_comment.md

```diff
@@ -0,0 +1,46 @@
+# Request for comment (RFC)
+
+## Why RFCs?
+This is part of the SilverStripe core decision-making process and addresses the gap between the idea phase and the pull request submission (and merge).
+
+The rationale behind this process is to:
+ * Improve visibility on how the decisions made, producing a more open community mood
+ * Clarity on what the proposal is, its rationale and impact
+ * Reduce unnecessary work when coding large features is done without community and core-committers buy-in
+ * Improved likelihood of an optimal solution being merged - the pressure is much higher to approve the change if the coding has already been done (it’s much easier to make adjustments in the design phase)
+
+The important thing to understand about the RFCs is that these are NOT a way to request features. Rather, they are a way to bring clarity so people understand what change is being proposed.
```

Comment:
Maybe add here what we've been saying all along - if you bring an RFC up, it's also assumed that you are proposing to develop it and have necessary resources available at some point in the future :-)